interface/ceed.c

```diff
@@ -335,6 +335,27 @@ int CeedInit(const char *resource, Ceed *ceed) {
   return 0;
 }
 
+/**
+  @brief Retrieve a delegate CEED
+
+  @param ceed Address of Ceed to retrieve delegate of,
+              address to save delegate to if one exists
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Utility
+**/
+int CeedGetDelegate(Ceed *ceed) {
```

Comment:
Overwriting the input argument is weird. What if the caller doesn't want to overwrite?

Reply:
Fixed